### ISSUE TITLE
Dropdown is Duplicating in MVC

### ISSUE THREAD

**Issue by rberger247** [davidstutz/bootstrap-multiselect#1170]
The dropdown is duplicating instead of replacing the default dropdown.

<img width="254" alt="multiselect1" src="https://user-images.githubusercontent.com/29547972/103803108-1ebee500-501e-11eb-815c-5175c20fa74e.PNG">

<img width="289" alt="multiselect2" src="https://user-images.githubusercontent.com/29547972/103803126-27172000-501e-11eb-8890-60a4d719b9dd.PNG">

`@html.listBoxFor(m => m.busrule[i].task, new multiselectList(slh.getRegAns), "value", text, m.busrule[i].busRulans, new {@class= "listbox"})`
```
$('.listbox').multiselect({
includeSelectAllOptions:true;
})
```

Notes:
-  code is shortened for concision
- -  Using boostrap v3 
- -  Using Asp.Net 4.5
- -  Select2 is in the project but select2 is not being utilized for the specific control

**Comment by tiesont** [comment]
Have you included the bootstrap-multiselect.css (or min version) file? Would be helpful if your example was more complete - I've used this plugin in an ASP.NET MVC project a few times with no issues, so the problem is something that you're not showing.

**Comment by rberger247** [comment]
Yes, it is included and is displayed as a source in the developer tools. Is there something in particular that is not complete? It is concise but not sure what other details are missing.

**Comment by tiesont** [comment]
I can't reproduce what you're seeing, from your example. Your screenshot seems to imply that you're rendering this in an accordion or similar - that's relevant, as Bootstrap dropdowns (in v3) generally have issues being used in a collapse section (https://github.com/davidstutz/bootstrap-multiselect/issues/180). 

The root issue seems to be related to trying to apply positioning rules to elements that aren't actually visible, if I recall correctly. You may want to try initializing the dropdown in each section when the section is toggled, rather than on page load / DOM ready.

**Comment by WOPiersol** [comment]
I am having this issue as well.

I am using the following:
- jQuery JavaScript Library v3.3.1
- Bootstrap v4.3.1
- .NET Core 3.1

HTML:
```
<!-- Build your select: -->
<select` id="example-getting-started" multiple="multiple">
    <option value="cheese">Cheese</option>
    <option value="tomatoes">Tomatoes</option>
    <option value="mozarella">Mozzarella</option>
    <option value="mushrooms">Mushrooms</option>
    <option value="pepperoni">Pepperoni</option>
    <option value="onions">Onions</option>
</select>
```

Jquery:
```
        $(document).ready(function () {

            $('#example-getting-started').multiselect();

        });
```
CSS Loaded in Layout page
![image](https://user-images.githubusercontent.com/78929260/107684776-cc01ca00-6c68-11eb-8c44-4b6456973417.png)

JQuery Loaded in Layout page
![image](https://user-images.githubusercontent.com/78929260/107684913-f784b480-6c68-11eb-8505-d3cb56671907.png)



![image](https://user-images.githubusercontent.com/78929260/107684376-5269dc00-6c68-11eb-9fcb-4f8870575646.png)

**Comment by WOPiersol** [comment]
I was resolve this issue by simple adding a style of display:none; to the original select.  

```
<!-- Build your select: -->
<select id="example-getting-started" multiple="multiple" class="form-control" style="display:none;">
    <option value="cheese">Cheese</option>
    <option value="tomatoes">Tomatoes</option>
    <option value="mozarella">Mozzarella</option>
    <option value="mushrooms">Mushrooms</option>
    <option value="pepperoni">Pepperoni</option>
    <option value="onions">Onions</option>
</select>
```

![image](https://user-images.githubusercontent.com/78929260/107996538-3d61b580-6fa6-11eb-8d0c-1657ce659e1f.png)

**Comment by tiesont** [comment]
This really shouldn't be necessary - the plugin's CSS already has rules that are supposed to (effectively) hide the `<select>` element. It's possible something changed in the generated HTML which has caused that rule to no longer be applied. Will need to investigating.

Worth mentioning that adding "display: none" can have some unintended consequences - some plugins will not select elements which are not displayed, so you might have problems selecting the... select... to which _this_ plugin has been applied.

**Comment by WOPiersol** [comment]
Yeah, it is not working correct with or without hiding the original select.  Another issue I am having it loading the selected options from a DB.  If I make a selection and save it, it loads fine.  But, on the original visit to the page, it does not load the selected options.  I like the looks of the end result using it, but there are too many issues with this plug in.   I will be going back to the drawing board and looking for another solution.  I need something working now for the project I am working on.  Maybe I will revisit when I have more time to debug.
